modules/core/src/system.cpp

```diff
@@ -597,6 +599,10 @@ struct HWFeatures
         have[CV_CPU_VSX3] = (CV_VSX3);
     #endif
 
+    #if defined __riscv && defined __riscv_vector
+        have[CV_CPU_RVV] = true;
```

Comment:
This check should be dynamic, otherwise dispatching does not make sense. Is it possible to detect vector extensions in runtime?

Reply:
RISC-V vector extension spec have not mentioned about runtime check up to now. I will try to ask them about it.